### PR TITLE
Cleanup webpack

### LINKED ISSUE #1171: Remove webpack build configs

### PR DESCRIPTION
**Pull request by baurine**
close #1171 

Remove the unused files and configs caused by replacing webpack by esbuild.

### CODE REVIEW

File: ui/.storybook/main.js [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-        break
-      }
-    }
-  }
-
-  return config
-}
-
-// ref: https://harrietryder.co.uk/blog/storybook-with-typscript-customize-cra/
-const custom = require('../config-overrides')
```
Comment (baurine) [inline ui/.storybook/main.js:22]: The storybook configs depend on webpack, so remove them first.

File: ui/package.json [hunk 2]
```diff
@@ -63,7 +63,6 @@
     "build": "gulp build",
     "fmt": "prettier --write .",
     "gen:browserlist": "gulp gen:browserlist",
```
Comment (breeswish) [inline ui/package.json:65]: Are we completely removing the Storybook?

**Review by breeswish — APPROVED** [review]

File: ui/package.json (hunk 2 above)
Comment (baurine) [reply ui/package.json:65]: Yep, it doesn't work anymore. We can bring it back when we really need it, I think.

Comment (baurine) [reply ui/package.json:65]: Create an issue #1190 to track it.

**Pull request merged**